README.md

```diff
@@ -676,6 +678,33 @@ As a convenience, the `id` and `itemToString` props which you pass to
 
 ## Event Handlers
 
+Downshift has a few events for which it provides implicit handlers. Several of
+these handlers call `event.preventDefault()`. Their additional functionality is
+described below.
+
+### default handlers
+
+* `ArrowDown`: moves the highlighted index down by 1. If this shift key is held
+  when this event fires, the highlighted index will jump down 5 indices instead of 1.
+  NOTE: if the current highlighed index is within the bottom 4 indices, the top-most
+  index will be highlighted.)
+
+* `ArrowUp`: moves the highlighted index up by 1. If this shift key is held when
+  this event fires, the highlighted index will jump up 5 indices instead of 1. NOTE:
+  if the current highlighed index is within the top 4 indices, the bottom-most index
+  will be highlighted.)
+
+* `Enter`: if the menu is open, select the currently highlighted item. If the menu
+  is open, the usual 'Enter' event is prevented by Downshift's default implicit enter
+  handler; so, for example, a form submission event will not work as one might expect.
```

Comment:
Please add at the end of this sentence `(though if the menu is closed the form submission will work normally)`

Reply:
Oh, I figured I had this caveat covered by changing the start of the sentence to "If the menu is open…". Would you prefer I remove that and add your suggestion to the end?

Reply:
Yeah, let's just make the addition I suggested. I feel like it could be ambiguous otherwise.